docs/glsl-fuzz-walkthrough.md

```diff
@@ -47,6 +47,43 @@ Add the following directories to your path:
 The `graphicsfuzz/` directory is the unzipped release.
 If building from source, this directory can be found at `graphicsfuzz/target/graphicsfuzz/`.
 
+### amber
+
+In March 2019 we depreciated our legacy vulkan worker to instead rely on
```

Comment:
depreciated -> deprecated

vulkan -> Vulkan

Reply:
Also: I suggest adding a link here to the legacy documentation.  We state clearly in that documentation that we don't plan to keep it up-to-date, but no harm in linking to it in case someone needs to find it.

Reply:
Done